dev/js/project.js

```diff
@@ -2,7 +2,42 @@
 //
 // Authors: Natalia Świerczek (swierczek.n@gmail.com)
 // Copyright Natalia Świerczek Portfolio © All Rights Reserved
-//
-// Project JavaScript code
-console.log('Project JS');
-// ...
+(function () {
+    console.log('Animations start');
+
+    // anime({
```

Comment:
In production code we should avoid of commented code blocks like that - to remove